tests/anagram-all-fail/expected_results.json

```
@@ -0,0 +1,48 @@
+{
+  "tests": [
+    {
+      "test_code": "    h.assert_array_eq[String](failme, Anagram(\"bobs\", anagrams))",
+      "message": " Assert EQ failed.  Expected ([len=1: failme]) == ([len=0: ])",
+      "status": "fail",
+      "version": 2,
+      "name": "    // Anagram should return an empty array if there are no anagrams"
```

Could you maybe strip the leading spaces and the `// ` part? The name is displayed in the UI to identify the test case that was run.

```suggestion
      "name": "Anagram should return an empty array if there are no anagrams"
```

---

Was on my list - wanted to test the integration first.  It's pulling the descriptions from the comment in front of each assert in the tests.

(I also need to go through all the examples to ensure that they will play nice with this test-runner too)